waltz-ng/client/ratings/rating-utils.js

```diff
@@ -134,38 +134,54 @@ export function indexRatingSchemes(schemes = []) {
 export function distinctRatingCodes(schemes = {}) {
     return _.chain(schemes)
         .flatMap((v,k) => v.ratings)
-        .map('rating')
+        .map("rating")
         .uniq()
         .value();
 }
 
 
 export function mkAuthoritativeRatingSchemeItems(displayNameService) {
-    const resolveName = k => displayNameService.lookup('AuthoritativenessRating', k);
+    const resolveName = k => displayNameService.lookup("AuthoritativenessRating", k);
     return {
-        'DISCOURAGED': {
-            rating: 'R',
-            name: resolveName('DISCOURAGED'),
+        "DISCOURAGED": {
+            rating: "R",
+            name: resolveName("DISCOURAGED"),
             position: 30,
-            color: '#DA524B'
-       },
-       'SECONDARY': {
-            rating: 'A',
-            name: resolveName('SECONDARY'),
+            color: "#DA524B"
+        },
+        "SECONDARY": {
+            rating: "A",
+            name: resolveName("SECONDARY"),
             position: 20,
-            color: '#D9923F'
-       },
-       'PRIMARY': {
-            rating: 'G',
-            name: resolveName('PRIMARY'),
+            color: "#D9923F"
+        },
+        "PRIMARY": {
+            rating: "G",
+            name: resolveName("PRIMARY"),
             position: 10,
-            color: '#5BB65D'
+            color: "#5BB65D"
         },
-       'NO_OPINION': {
-            rating: 'Z',
-            name: resolveName('NO_OPINION'),
+        "NO_OPINION": {
+            rating: "Z",
+            name: resolveName("NO_OPINION"),
             position: 40,
-            color: '#939393'
+            color: "#939393"
         }
     };
-}
\ No newline at end of file
+}
+
+
+export function getDefaultRating(ratings = []) {
+    const defaultRating = _
+        .chain(ratings)
+        .sortBy(["position"])
+        .head()
+        .value();
+
+    const fallback = () => {
+        console.log("Could not determine default rating, using hard-coded [G]", { givenRatings: ratings });
+        return "G";
+    };
+
+    return _.get(defaultRating, ["rating"], fallback());
```

Comment:
Fix is here: getting the "rating" from the object.  
Everything else in this PR is tidying up.